receiver/awsfirehosereceiver/testdata/otlpv1_config.yaml

```diff
@@ -0,0 +1,7 @@
+awsfirehose:
```

Comment:
Is there a test using this file?

Reply:
Now there is! https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/36125/commits/5fca96bc0029e7b2f455a5141a9a5c79838b0cb8